common/src/main/cpp/include/classes/openxr_fb_hand_tracking_mesh.h

```diff
@@ -39,7 +39,11 @@ namespace godot {
 class OpenXRFbHandTrackingMesh : public Node3D {
 	GDCLASS(OpenXRFbHandTrackingMesh, Node3D)
 public:
-	using Hand = XRHandTracker::Hand;
+	enum Hand {
```

Comment:
Can't we instead use `XRPositionalTracker::TrackerHand` instead of introducing new enums?

Reply:
No, because the point of this enum (and the reason we originally had the 2nd enum in `XRHandTracker`) is that the values line up with the indexes used for the hands. So, this exists to give a name to the magic numbers we're using, that is 0 = left and 1 = right.

But I suppose we could use the custom enum only internally, and then use `XRPositionalTracker::TrackerHand` externally?

Reply:
Indeed - the code in this repository was using `XRHandTracker::Hand` as both container-size (HAND_MAX=2) and then indexing into that container (HAND_LEFT=0, HAND_RIGHT=1). `XRPositionalTracker::TrackerHand` does not work for this usage.

The introduction of `OpenXRFbHandTrackingMesh::Hand` is an attempt to cause the least amount of code-disruption while retiring the old (soon-to-be-deleted) enum. Switching to `XRPositionalTracker::TrackerHand` would necessitate changing the implementation from container-size / container-index and cause further changes in code which is already being refactored for XRTracker scene-tree changes.

Reply:
I would suggest after merging this, that https://github.com/GodotVR/godot_openxr_vendors/pull/131 be rebased; then subsequent work on normalizing the hand mesh could be performed on that PR as desired.

Reply:
@m4gr3d Actually in reviewing https://github.com/GodotVR/godot_openxr_vendors/pull/131 I'm actually thinking it might be better to keep it with a custom enum rather than using `XRPositionalTracker::TrackerHand`. Currently if a user manipulates the hand property they get:

![image](https://github.com/GodotVR/godot_openxr_vendors/assets/1863707/dfe1373c-09cf-48cd-be30-1e0ce7f81501)

If this were switched to the `XRPositionalTracker::TrackerHand` enum they would get an `Unknown` option - something not relevant for a hand-tracking-mesh.

Reply:
Sounds good!